sdk/eventhub/azure-eventhubs/azure/eventhub/extensions/__init__.py

```diff
@@ -0,0 +1,5 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)  # type: ignore
```

Comment:
@johanste this file ideally should be empty, unless we want easy "editable" mode and installed extensions at the same time

Reply:
If this init file in azure-eventhubs is empty, should the one in the plugin under the same folder also be empty?

Reply:
Updated